buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -719,7 +719,7 @@ public final class {{class}} {
   /**
    * Immediate client peer port number.
    *
-   * @deprecated This item has been renamed in 1.22.0 of the semantic conventions. Use {@link SemanticAttributes#NETWORK_PEER_PORT} on server telemetry and {@link SemanticAttributes#NETWORK_LOCAL_PORT} on client telemetry instead.
+   * @deprecated This item has been renamed in 1.22.0 of the semantic conventions. Use {@link SemanticAttributes#NETWORK_LOCAL_PORT} for local network telemetry and {@link SemanticAttributes#NETWORK_PEER_PORT} for peer network telemetry instead.
```

Comment:
I'm not sure I follow this, since you can have `network.local.port` and `network.peer.port` attributes both on the same telemetry item (span).